wfl/generate/minim.py

```diff
@@ -7,7 +7,7 @@
 from ase.constraints import ExpCellFilter
```

Comment:
Do you think `optimize.py` (following ASE) would be ever so slightly clearer than `minim.py`?

Reply:
For the sake of compatibility, I suppose that's fine.